tests/sentry/models/test_user.py

```diff
@@ -387,6 +393,8 @@ def test_only_source_user_is_member_of_organization(self, expected_models: list[
         RuleActivity,
         RuleSnooze,
         SavedSearch,
+        SpanAttributeExtractionRuleConfig,
```

Comment:
To pass this test, and the one above, you'll need to add this model to the list at `src/sentry/services/hybrid_cloud/organization/impl.py#559`. If you want to make this sharp edge a little easier to handle for future editors of this code, it'd be nice if you edit the test failure message (which I agree is currently quite uninformative!) to explain what went wrong and (the expected list of merged models did not much the ones that were actually merged).

Reply:
Thank you, this worked - I'll add a more verbose error message to those two tests indicating how to solve them.